### ISSUE TITLE
If change a short message to a long message causing line break dialog bugs

### ISSUE THREAD

**Issue by vssantoss** [jtsage/jquery-mobile-simpledialog#26]
Here am I again. :)

If you click on test 1 button on link below, the bug occurs on third message.

http://jsfiddle.net/vehuel/pjy3g/

This page load your cdn
- http://dev.jtsage.com/cdn/simpledialog/latest/jquery.mobile.simpledialog.min.js
- http://dev.jtsage.com/cdn/simpledialog/latest/jquery.mobile.simpledialog.min.css

Thank you.

**Comment by jtsage** [comment]
well.  isn't that odd.  Ok, I'll see what I can figure out.

**Comment by vssantoss** [comment]
Another detail. The results are different in firefox and chrome.

Thank you.

**Comment by jtsage** [comment]
Just another detail:  this happens on iPad too.  (IOS5)

**Comment by jtsage** [comment]
Ok, well.  First - the good news:  problem fixed.  http://jsfiddle.net/pjy3g/4/

It seems that I needed to go back in, set the width to auto immediately before calculating the expected width, then I went one step further and actually forced the content to be that width.  I don't do this in any other place than when the dialog is forcibly refreshed - seems to work with your example, lemme know if I've introduced any new problems by fixing it like this.
